### DISCUSSION TITLE
Gibt es Leute die nach VICTRON 150/35 suchen?

### DISCUSSION THREAD

**Discussion by Manos1966** [hoylabs/OpenDTU-OnBattery#530] [General]
Rein Informativ, AMAZON Black Friday Angebot fuer dein VICTRON MPPT Smart 150/35 liegt bei 208EUR 
Und fuer 11EUR extra, kann man einen dreijaehrigen Geraeteschutz addieren (Schutz gegen Unfallschäden sowie Deckung von Pannen und Fehlern, die nicht von der Verkäufer-/Herstellergarantie abgedeckt sind).

Vorsichtshalber: ich verdiene nichts und habe keine Beziehung zu der Firma die das anbietet. Falls dieser Post gegen Richtlinien dieses Forums ist, bitte Bescheid sagen und ich werde es sofort löschen.

**Comment by madmartin** [comment]
Hi, rein informativ ist das kein gutes Angebot, abgesehen davon daß ich es nicht finden kann. Benutzt Preisvergleich-Webseiten wie idealo.de oder geizhals.de um gute Angebote zu finden. Aktuell gibts den SmartSolar MPPT 150/35 als günstigstes Angebot für 141,14 €. Wenn Euch dann das Angebot zu günstig erscheint prüft Ihr noch mit dem https://www.verbraucherzentrale.de/fakeshopfinder-71560 ob es Anzeichen für einen Fake-Shop gibt. Das ist bei dem 141€ Angebot nicht der Fall.

**Reply by Manos1966** [reply]
WOW! 145EUR ist ein verdammt guter Preis! Danke fuer den Tipp

Stimmt, der Preis von 208EUR ist jetzt 237EUR!
![Manos AMAZON Black Friday - Copy](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/6bb23be0-9d21-4bc4-bea5-b7f57f123033)

**Reply by peff74** [reply]
Verkauf Bochum ist vertrauenswürdig, habe mir dort nich einen 100/20 geholt. Hatten die auch zum allzeit Tiefpreis.
Ist heute angekommen^^.

**Reply by peff74** [reply]
und nimmste 100/20 oder 150/35?

Ich spiele noch mit dem Gedanken einen 150/35 mal zu testen.
Wir verschaltet man die am besten?
Je 2 Paneele in Reihe dann Parallel?

**Reply by Manos1966** [reply]
Danke @madmartin und @peff74 ohne euer Input hätte ich nicht gewagt, etwas da zu bestellen 👍 

Die Tatsache, dass ich sogar mit AMAZON PAY zahlen konnte, war der letzte Grund, warum ich meine Bestellung platziert habe.

Vorausgesetzt, ein MPPT Smart 150/35 tatsächlich kommt, habe ich 65EUR gespart 💯 

@peff74 Victron 150/35 weil es sieht so aus, als ob der 100/20 (den ich am Anfang haben wollte) Probleme haben beim Sonne/Wolken Wechsel 

(es gibt eine Post darüber im Forum - sag falls du sie nicht findest) <- Quatsch!, Du warst derjenige der damals gepostet hat :-D

**Reply by spcqike** [reply]
> Wir verschaltet man die am besten?
> Je 2 Paneele in Reihe dann Parallel?

kommt auf die Paneele an, würde ich sagen :D aber mit gängigen 54-72 Zellen und 350-450W: ja. 
so wie hier
https://github.com/helgeerbe/OpenDTU-OnBattery/discussions/249

**Reply by Manos1966** [reply]
Ich wollte drei in Reihe und danach parallel anschließen, bis ich auf die Temperatur Koeffizient (Voc) gestossen bin :-(
Daher nur zwei in Reihe.

**Reply by peff74** [reply]
also bei dem Preis habe ich jetzt auch noch einen 150/35 geordert^^

Für den 100/20 findet sich sicher ein Abnehmer und ich ärgere mich nicht mehr über die Aussetzer.

**Reply by pelletq** [reply]
Hi, redest du vom 150/35 oder vom 100/20?
Ich beginne gerade damit, meine Anlage zu planen/zu besorgen und hatte auch vor 2x 3 Module in Reihe an den 150/35 anzuschließen (also 3 jeweils in Reihe und dann die beiden Strings parallel). Laut Victron-Rechner (der ja auch Temp-Koeff. berücksichtigt, passt das mit einer Überbelegung unter 130%.
Welche Module hattest du geplant, bzw. welche Kennwerte hatten diese, dass der 150/35 dafür nicht ausreicht?

**Comment by Manos1966** [comment]
Hallo @pelletq Erinnere mich, dich zum Kaffee einzuladen!

Ich verstehe nicht ganz warum, ich sehe einen deutlichen Unterschied zwischen meine Berechnungen und der Daten des Victron MPTT Calculators! :-) 💯

**Reply by Manos1966** [reply]
Das sind meine Panels.
![EckDaten](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/ee66578b-2a70-49e4-9680-08596b8691a9)

Laut MPTT Calculator, bei 3 Paneele Serie, sogar bei minus 20 C, uebersteigt die Spannung **nicht** die 140V 🥰 
![3S2P 01](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/c4b00936-df98-42fe-9870-a0bc4b7dae6e)
![3S2P 02](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/42a0cc24-ea28-4908-9733-c5f4582307e1)

**Reply by Manos1966** [reply]
added it to the Wiki

[Victron MPPT battery charger](https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Victron-MPPT-battery-charger)

**Comment by Manos1966** [comment]
Update @madmartin und @peff74

mein Victron MPTT 150/35 wurde heute geliefert 💯 

PERFEKT!

Vielen herzlichen Dank @madmartin , das war wirklich ein Super Tipp!! 🥇

**Reply by madmartin** [reply]
Hi,
ich habe einen Victron Smartsolar 150/45 bestellt und heute nach knapp 2 Wochen geliefert bekommen! Wuchtiges Teil!
Und für 166,66€ ein echtes Schnäppchen. Verkauf-bochum punkt DE scheint also ein reeller Versand zu sein.

**Reply by pelletq** [reply]
Eine Alternative ist auch noch fraron de 
Dort habe ich mir den 150/35 geholt. War durchweg günstiger als verkauf-bochum und comkor, ist aber irgendwie nicht auf idealo gelistet.

Also so langsam sammel ich die Komponenten ein und kann bald mal näher in die Planung und Umsetzung gehen

**Comment by Manos1966** [comment]
heute wurde mein zweiter Victron MPTT 150/35 geliefert, mit 5 Tage Lieferzeit, sogar schneller als der Erste.
Es sind echte Victrons (Sorry, bin misstrauischer Typ, "Glaube nichts von dem, was du hörst, und nur die Hälfte von dem, was du siehst"), sogar der (entsetzliche) Geruch der Verpackung ist gleich wie bei meiner Amazon Bestellung. 

Der Laden verdient 4 Sterne und bessere Bewertungen.... ohne @madmartin und @peff74 Meinung gelesen zu haben, haette ich nicht von da bestellt.

Jetzt muss ich wieder @madmartin fragen 🙈  
Ist der Preis von 1EUR pro Meter fuer 6mm2 Solar Kabel H1Z2Z2-K ein guter Preis, oder muss ich anderweitig suchen?   
(ich brauche 100 Meter - am liebsten 50m Rot / 50m Schwarz)

**Reply by madmartin** [reply]
> Ist der Preis von 1EUR pro Meter fuer 6mm2 Solar Kabel H1Z2Z2-K ein guter Preis, oder muss ich anderweitig suchen? (ich brauche 100 Meter - am liebsten 50m Rot / 50m Schwarz)

Alle Händler die ich mal fix abgeklappert habe sind deutlich teurer als 50m für 50€. Wenn es ein seriöses Angebot ist dann zugreifen.
Ich kann nur den Händler empfehlen wo ich schon mehrmals gekauft hab und die Qualität okay war und der Preis auch - das ist der Verkäufer "Elektriese_eu" bei ebay. Ich hab da in der Vergangenheit festgestellt daß die fertigen Kabel mit Steckern unschlagbar günstig sind, sogar günstiger als reine Meterware. 2x50m 6mm² je 1x rot und schwarz kosten 141€ - allerdings mit MwSt und die kann man leider bei ebay nirgends abwählen.

**Reply by Manos1966** [reply]
Hallo @madmartin 

ich habe das Kabel heute abgeholt. Es scheint original zu sein.
KBE SOLAR DB+ 6,00 MM2 H1Z2Z2-K 62930 IBC 131 PV 1500-K HALOGEN FREE LOW SMOKE MADE IN GERMANY
![Bought 6mm2 cable](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/b29b85bd-905d-4283-b23c-f612e31867c3)

**Comment by Manos1966** [comment]
Hi @madmartin 

Hast du einen guten Vorschlag fuer Bezugsquellen fuer Erdungskabel?
Meine Pylontech US5000 braucht AWG 6 (16mm2) Erdkabel 🤦‍♂️ 


(AWG 4 Fehler wurde korrigiert)

**Reply by Tiese** [reply]
Hallo @Manos1966,
da stimmt aber was nicht, bei deinen Angaben...
Laut [https://www.reichelt.de/reicheltpedia/index.php/AWG](https://www.reichelt.de/reicheltpedia/index.php/AWG) ist 4 AWG 25 mm² und laut US5000 Bedienungsanleitung ([https://mitzner-energie.de/datenblat/US5000_Bedienungsanleitung_DE_V1.0.pdf](https://mitzner-energie.de/datenblat/US5000_Bedienungsanleitung_DE_V1.0.pdf) - Seite 24) wird ein Erdungskabel mit mind. 10 AWG, also mind. 6 mm² (welches im Lieferumfang ist) benötigt, was sich auch [hiermit](https://www.elektropraktiker.de/nachrichten/nachricht/querschnitt-der-schutzpotentialausgleichsleiter?p=all#:~:text=544.1%20Schutzpotentialausgleichsleiter%20für%20die%20Verbindung,50%20mm2%20Stahl.) deckt!

**Comment by Manos1966** [comment]
Danke @Tiese 
Entschuldigung, ich habe mich vertan :(  
6 AWG (16mm2) war gemeint.
Interessant ist auch, meine zwei US5000 sind mit 6 AWG geliefert.
Und meine Version des Handbuchs (Information Version: PM0MUS500255 /  SD21US501001) sprich von 6 AWG
![Pylontech Cables delivered](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/c1a2a563-ffc2-4409-bb75-67e4b664030c)

**Reply by Tiese** [reply]
Ich habe die US3000C (leider noch nicht fertig angeschlossen) und da ist ein 10 AWG-Kabel dabei.

**Reply by Manos1966** [reply]
Ja, das ist richtig 10 AWG ist bei den US3000

Ich habe noch etwas bemerkt. Die deutsche Anleitung hat Fehler! 
Sie zeigt, dass die US5000 seitlich platziert werden koennen.
Im Gegenteil, in meiner englischen Anleitung ist es nicht erlaubt!
![Untitled](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/0984e1f7-3db7-4c6f-8189-89c5edae5b89)

[AU-US5000+user+manual.pdf](https://github.com/helgeerbe/OpenDTU-OnBattery/files/13953364/AU-US5000%2Buser%2Bmanual.pdf)

**Reply by Tiese** [reply]
Merkwürdig &#129300;

**Comment by knopers1** [comment]
Moin,
ich betreibe zur Zeit den Victron 100/20 mit zwei 400 Watt Modulen in Reihe und würde gerne umsteigen auf Victron 150/35. 
Dazu würde ich gerne ein drittes Modul in Reihe schalten. Der Laderegler ist an einem Pylontech 48V angeschlossen. 

Läuft bei Euch der Victron ohne Probleme? Aussetzer bei bewölktem Himmel?
Muss der Drehschalter auf die Position 7 eingestellt sein? Beim Victron 100/20 gibt es diesen Drehschalter nicht.

**Reply by Snusme83** [reply]
Servus, das wäre bei mir auch zu überlegen umzustellen auf 2 x Victron 150/35 anstatt 3 x Victron 100/20! Das würde dann zwei Probleme lösen. 
- Die Aussetzer bei Bewölkung der Victron 100/20.
- Falls OpenDTU-on Battery keine 3 Laderegler unterstützen soll!

Haben 6 x Module mit 370 Wp, 2 x Pylontech US3000C, 3 x Victron Smartsolar 100/20 und einen Hoymiles HM 1500!
